src/autocapture.js

```diff
@@ -221,6 +221,14 @@ var autocapture = {
 
         if (!this._maybeLoadEditor(instance)) { // don't autocapture actions when the editor is enabled
             var parseDecideResponse = _.bind(function(response) {
+                if(response['isAuthenticated'] && response['toolbarVersion'].indexOf('toolbar') === 0) {
+                    this._loadEditor(instance, {
+                        apiURL: instance.get_config('api_host'),
+                        jsURL: response['jsURL'] || instance.get_config('api_host'),
```

Comment:
What's the difference between jsURL and api_host/apiURL?

Reply:
when `settings.DEBUG`, `jsURL` will be `http://localhost:8234` - the HMR url

Reply:
Okay makes sense